google/cloud/aiplatform/training_jobs.py

```diff
@@ -2918,10 +2917,18 @@ def _run(
 
         training_task_definition = schema.training_job.definition.automl_tabular
 
+        if self._column_transformations is None:
+            column_transformations = [
+                {"auto": {"column_name": column_name}}
+                for column_name in dataset.column_names()
```

Comment:
I wanted to use just `column_names` instead of `column_names()` because it's a property but I couldn't get the unit-test mock working for the life of me.

Reply:
Fixed it.